### DISCUSSION TITLE
SSO processing issue

### DISCUSSION THREAD

**Discussion by stackcanvas** [supabase/supabase#27084] [Questions]
Hello, hope you are all good.

I have below issue after saml sso integration on my application.

{"code":400,"error_code":"saml_assertion_no_email","msg":"SAML Assertion does not contain an email address"}

Hope someone helps me out.

Thanks

**Comment by opgbaudouin** [comment]
Although I'm new to this - the SSO documentation clearly states you MUST return an Attribute that can be considered an emailaddress. 

https://supabase.com/docs/guides/auth/enterprise-sso/auth-sso-saml?queryGroups=language&language=js#why-do-some-users-get-saml-assertion-does-not-contain-email-address

So i presume your SSO / SAML providers needs to provide this field
